packages/app/chrome/test/chrome.test.ts

```diff
@@ -0,0 +1,34 @@
+import { resolve } from "path";
+import * as puppeteer from "puppeteer";
+import { TestServer } from "../../common/test";
+
+describe("chrome e2e", () => {
+	const server = new TestServer({ auth: false });
+	let browser: puppeteer.Browser;
+	let page: puppeteer.Page;
+	beforeAll(async () => {
+		await server.killProcesses();
+		server.start();
+		browser = await puppeteer.launch();
+	});
+	beforeEach(async () => {
+		page = await browser.newPage();
+	});
+	afterAll(async () => {
+		await browser.close();
+		server.dispose();
+		await server.killProcesses();
+	});
+
+	server.test("should open IDE", async () => {
```

Comment:
I'd use normal `it` here if possible. Will assist in readability.

Reply:
My thought process is that:
```
server.test("should do a thing", async () => {
    await doThing();
}, 1000);
```
Is prettier than:
```
it("should do a thing", async () => {
    try {
        await doThing();
    } catch (e) {
        // make sure the server closes itself, if the thing doesn't work
        await this.dispose();
        throw e;
    }
}, 1000);
```

Reply:
Could you dispose the server in `afterEach`?

Reply:
Yes, that would. I've made changes so that's no longer necessary. The tests are using `it` now.